wdl-lint/tests/lints/import-sorting/source.errors

```diff
@@ -0,0 +1,12 @@
+note[ImportSort]: imports are not sorted lexicographically
+  ┌─ tests/lints/import-sorting/source.wdl:7:1
+  │
+7 │ import "A"
```

Comment:
@a-frantz - Can we have a zero-length highlight? It feels odd to have `^^^^^^` highlighting the `import` keyword.

Reply:
@adthrasher I'm happy with this 👍

Reply:
LGTM!